Habitica/src/main/java/com/habitrpg/android/habitica/ui/viewHolders/tasks/BaseTaskViewHolder.java

```diff
@@ -87,14 +97,26 @@ public void bindHolder(Task newTask, int position) {
         if (this.rightBorderView != null) {
             this.rightBorderView.setBackgroundResource(this.task.getLightTaskColor());
         }
+	    if (task.getTags() != null && !task.tags.isEmpty())
+		    this.tagsGlyph.setVisibility(View.VISIBLE);
     }
 
+	@OnClick(R.id.tags_glyph)
+	void displayTagsToast(){
+		String tagString = "";
+		for(TaskTag t:
+		    task.getTags())
+		{
+			tagString = tagString + t.getTag().getName() + ", ";
+		}
+//		Toast.makeText(context, tagString, Toast.LENGTH_LONG).show();
```

Comment:
Yes i meant to leave this. In #569 I explained why it is there. Some how there are no name strings in the tag objects only the tag IDs. So I wanted to wait for a hint on how to get the names properly and not write a workaround or something similar. It is meant to tell the user which tags are on the task.